apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -11,12 +11,11 @@ import {
 } from '@aragon/ui'
 import { FormField, FieldTitle } from '../../Form'
 import useGithubAuth from '../../../hooks/useGithubAuth'
+import useSingleIssue from '../../../hooks/useSingleIssue'
```

Comment:
Maybe since the panels don't work with routing yet anyhow, we can revert all changes to these panel files, which could even let us remove `useSingleIssue` for now. I'd rather figure out patterns we like for core routing before committing to fuller tooling. Maybe we'll settle on a pattern that turns out to not need `useSingleIssue`.